scripts/performance/run-performace-tests-functions.sh

```diff
@@ -92,7 +95,8 @@ function update_setup_confiuration() {
 function execute_perfpro() {
     yum install ansible -y
     pushd $SCRIPT_LOCATION/performance/PerfPro
-    ansible-playbook perfpro.yml -i inventories/hosts --extra-vars '{ "EXECUTION_TYPE" : "sanity" ,"REPOSITORY":{"motr":"cortx-motr","rgw":"cortx-rgw"} , "COMMIT_ID": { "main" : "d1234c" , "dev" : "a5678b"},"PR_ID" : "cortx-rgw/1234" , "USER":"Shailesh Vaidya","GID" : "729494" }' -v
+    add_primary_separator "Executing Ansible CLI"
+    ANSIBLE_LOG_PATH=$ANIBLE_LOG_FILE ansible-playbook perfpro.yml -i inventories/hosts --extra-vars '{ "EXECUTION_TYPE" : "sanity" ,"REPOSITORY":{"motr":"cortx-motr","rgw":"cortx-rgw"} , "COMMIT_ID": { "main" : "d1234c" , "dev" : "a5678b"},"PR_ID" : "cortx-rgw/1234" , "USER":"Shailesh Vaidya","GID" : "729494" }' -v
```

Comment:
Are these vars going to be hardcoded?

Reply:
@shailesh-vaidya, can you answer this?

Reply:
Removed hardcoded values. Test job is in-progress - https://eos-jenkins.colo.seagate.com/job/Cortx-Automation/job/Performance/job/run-performance-sanity/127/console

Reply:
Job is successful now. Please review